src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work!");
```

Comment:
The message should indicate what type of `Machine` is working right now. Consider including the specific type like 'Truck' in the message to make it more informative.